src/components/ResourceList/README.md

```diff
@@ -507,7 +507,8 @@ function ResourceListWithFilteringExample() {
     [],
   );
   const handleQueryValueChange = useCallback(
-    (value) => setQueryValue(value),
+    (value) =>
```

Comment:
Somewhat unrelated but `setQueryValue` is already a stable function (i.e. it doesn't get redefined on each rerender), so you don't need to wrap it in a useCallback.

Can you `const handleQueryValueChange = setQueryValue;` or remove this intermediate variable in the first place?

Reply:
Will do, this is mostly copied from another example so I'll update it there too 👍